tests/console/postgresql94server.pm

```diff
@@ -17,18 +17,15 @@ sub run() {
     become_root;
 
     # install the postgresql94 server package
-    script_run "zypper -n in postgresql94-server && echo 'postgresql94-server_installed' > /dev/$serialdev";
-    die "postgresql94-server install failed" unless wait_serial "postgresql94-server_installed", 200;
+    assert_script_run "zypper -n in postgresql94-server", 200;
```

Comment:
jein - for some historic reason this function installs it through xterm - which is not working well in console tests.